packages/dashboard/src/components/widgets/tile/tile.tsx

```diff
@@ -23,7 +23,6 @@ import './tile.css';
 import { onChangeDashboardGridEnabledAction } from '~/store/actions';
 import { CSVDownloadButton } from '~/components/csvDownloadButton';
 import { StyledSiteWiseQueryConfig } from '~/customization/widgets/types';
-import { useGetConfigValue } from '@iot-app-kit/react-components';
```

Comment:
you need it remove it from the getConfigValue as well ?

Reply:
done! removed tthis and echarts flag since they're no longer gated